store/auction.ts

```diff
@@ -0,0 +1,46 @@
+import { actionTree } from 'typed-vuex'
+import { AuctionModuleState } from '@injectivelabs/chain-consumer'
+import { fetchAuctionModuleState } from '~/app/services/auction'
+
+// hard code auction round to optimize performance by skipping fetchAuction api call if auctionsViewed contains auctionRound
+export const hardcodedAuctionRound = 1
+export const hardcodedEndTime = 1639573200 * 1000
```

Comment:
Remove these, no need to skip `fetchAuction`. We can skip calling this function if the flag `SHOW_AUCTION_COUNTDOWN` is set to true